sylvia/src/types.rs

```diff
@@ -434,6 +434,19 @@ impl<'a, Contract: ?Sized> Remote<'a, Contract> {
     pub fn executor(&self) -> ExecutorBuilder<(EmptyExecutorBuilderState, Contract)> {
         ExecutorBuilder::<(EmptyExecutorBuilderState, Contract)>::new(&self.addr)
     }
+
+    pub fn update_admin(&self, new_admin: String) -> WasmMsg {
```

Comment:
Consider changing this signature to `pub fn update-admin(&self, new_admin: impl FromStr) -> WasmMsg`. This way it would be more convenient to use with string slices (which you do in tests).  It is, in general, a good habit to avoid `String` as an argument - it is typically better to either go with `&str`/`iml FromStr` (depending if we would make use of ownership).

Reply:
Good point!